spec/features/visitor_can_view_city_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature "Visitor can view a specific city show page" do
+  scenario "when they select a city from the main page dropdown" do
+    city = create(:city_with_homes)
```

Comment:
Nice use of factory girl. I didn't because I wanted to get a handle of what we're asking but will switch to factory girl afterward.